examples/entities.yaml

```diff
@@ -12,11 +12,13 @@ apiVersion: backstage.io/v1alpha1
 kind: Component
 metadata:
   name: example-website
+  namespace: default
   annotations:
     aws.amazon.com/aws-codepipeline-tags: component=example-website
     aws.amazon.com/aws-codebuild-project-tags: component=example-website
     aws.amazon.com/amazon-ecs-service-tags: component=example-website
     aws.amazon.com/cost-insights-tags: component=example-website
+    aws.amazon.com/aws-ecr-repository-arn: 1234567890.dkr.ecr.us-east-1.amazonaws.com/example-website
```

Comment:
Hmmmm, I want this to work with this repo and the example CDK without any modification as all the other examples do.

Think we have two options:
1. Add back in an annotation `aws-ecr-repository-name`, which doesn't have an equivalent in the other plugins
2. Support tags. The backend code for this is relatively simple because we've abstracted the hard work. The frontend gets annoying because you have to be able to handle potentially getting multiple repositories back.